commerce-lang/src/main/resources/content/Language.properties

```diff
@@ -1,6 +1,8 @@
 a-new-password-will-be-sent-to-x-if-you-can-correctly-answer-the-following-question=A new password will be sent to {0} if you can correctly answer the following question.
 accessories=Accessories
 account-groups=Account Groups
+account_role_administrator=Account Administrator
+account_role_order_manager=Account Order Manager
```

Comment:
Do these lang keys need to be underscore-delimited? They should be dash-delimited if at all possible.